sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala

```diff
@@ -324,4 +324,16 @@ class QueryExecutionErrorsSuite extends QueryTest
     assert(e5.getSqlState === "42000")
     assert(e5.getMessage === "Cannot parse decimal")
   }
+
+  test("CAST_CAUSES_OVERFLOW") {
```

Comment:
add the specific title like:
```suggestion
  test("CAST_CAUSES_OVERFLOW: from timestamp to int") {
```

Reply:
Done，@MaxGekk！Thanks